crates/agglayer-aggregator-notifier/src/settlement_client/rpc.rs

```diff
@@ -236,14 +233,15 @@ impl<StateStore, PendingStore, PerEpochStore, RollupManagerRpc>
 where
     RollupManagerRpc: L1TransactionFetcher,
     PerEpochStore: PerEpochWriter + PerEpochReader,
+    StateStore: StateWriter,
 {
     #[tracing::instrument(skip(self), fields(%settlement_tx_hash, %certificate_id))]
     async fn wait_for_settlement(
         &self,
         settlement_tx_hash: SettlementTxHash,
         certificate_id: CertificateId,
     ) -> Result<(EpochNumber, CertificateIndex), Error> {
-        info!(%settlement_tx_hash, "Waiting for settlement of tx {settlement_tx_hash}");
+        info!("Waiting for settlement");
```

Comment:
Why removing settlement tx hash from the message? It was very useful for inspections of the logs.

Reply:
You can maybe unify, and rename all `settlement_tx_hash` occurences to `tx_hash`.

Reply:
> Why removing settlement tx hash from the message? It was very useful for inspections of the logs.

It is still added as a structured log field by the `#[instrument]` annotation on the function. I thought the trend was to remove duplicate information from messages when that information is already provided in the structured format.

Having said that, I am happy to revert these changes as they are not super relevant to the substance of this PR.

Reply:
We should for sure add it to the spans, but not remove it from the messages. It is much easier to track logs this way in my experience. Alternative to have 10 `Waiting for settlement`and to open every one to see the details and the related hash is much less productive and harder to debug.

Reply:
So spans are for search and filtering, messages are to easily track whole log. Both should have enough info from my recent experience.

Reply:
> So spans are for search and filtering, messages are to easily track whole log. Both should have enough info from my recent experience.

Ok, re-added the certificate and transaction IDs where appropriate.